src/snowflake/cli/api/identifiers.py

```diff
@@ -62,7 +62,7 @@ def identifier(self) -> str:
 
     @property
     def url_identifier(self) -> str:
-        return ".".join(unquote_identifier(part) for part in self.identifier.split("."))
+        return ".".join(identifier_for_url(part) for part in self.identifier.split("."))
```

Comment:
@sfc-gh-turbaszek added a specialized version of `unquote_identifier` that performs `encodeURIComponent` on the unquoted identifier, and ported this fix to FQN so that other entity type can use it.